stories/Molecules/Navbar/Pagination/pagination.scss

```diff
@@ -0,0 +1,49 @@
+@import '../../../assets/scss/breakpoints';
+@import '../../../assets/scss/variables';
+
+/* Tab start */
+
+.pagination__wrapper {
+  align-items: center;
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+
+  .page__item {
```

Comment:
@harishkumar-srijan We can keep the page__item outside of pagination__wrapper class